src/mlpack/methods/adaboost/adaboost_main.cpp

```diff
@@ -203,7 +203,7 @@ void BINDING_FUNCTION(util::Params& params, util::Timers& timers)
       // Extract the labels as the last dimension of the training data.
       Log::Info << "Using the last dimension of training set as labels."
           << endl;
-      labelsIn = conv_to<Row<size_t>>::from(
+      labelsIn = arma::conv_to<arma::Row<size_t>>::from(
```

Comment:
@rcurtin this might look weird. However, I have no idea how the original expression was compiled with no error.
When I added the `conv_to` class the compiler complained about the above expression since it was ambiguous.

using `conv_to<arma::Row<size_t>>` is still ambiguous to the compiler, I think because there is a nested templates. The only way around this to make it conv_to explicitly called from armadillo.

The other solution would be to define a struct in arma_util, similar to the `UColType`, also I do not think this is necessary right now just for the sake of finishing the bandicoot integration in `ann`

Reply:
There's a `using namespace arma` in the file, which explains why it worked before.  Personally I would rename the `mlpack::conv_to` class to something different (say, `mlpack::ConvTo`), otherwise whenever a user does both `using namespace arma` and `using namespace mlpack`, they will have this ambiguity issue if they try to use `conv_to` anywhere.

Reply:
Okay, I see now, the change make sense actually, I will give it a try and see :+1: